### DISCUSSION TITLE
Welcome to modelica-buildings Discussions!

### DISCUSSION THREAD

**Discussion by mwetter** [lbl-srg/modelica-buildings#3105] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lambtt** [comment]
Howdy, guys!

I'm Wenqi from Texas A&M University. I'm now working on developing a heat pump system with thermal energy storage.
I have to say that Modelica Buildings Library (MBL) is amazingly great!
I could connect the models and check my idea very conveniently. 
As a user, MBL makes me start to wonder how to build a robust, fast, and accurate model if I'm a developer.
Now I'm reading some books about continuous system modeling and simulation to get some experience.
I hope I can be an expert in this field and hope the MBL community can grow bigger and bigger 💪!

**Reply by mwetter** [reply]
Welcome Wenqi. We wrote some guidance and experiences from developing models in the user guide at https://simulationresearch.lbl.gov/modelica/userGuide/index.html
I suggest you also familiarize yourself with the books mentioned at https://simulationresearch.lbl.gov/modelica/userGuide/gettingStarted.html#literature-for-users which are a great resource.

**Reply by lambtt** [reply]
Hey Michael, thanks for your suggestion. Those help me a lot to understand the Modelica language!

**Comment by NicholasFry** [comment]
Hi all,

I am a geothermal scientist with Egg Geo LLC. I am just beginning on MBL. The company is also hiring individuals with experience in MBL. https://www.linkedin.com/jobs/view/3436552834/?refId=iWJzZNhFRxyZCXWN5wOeQw%3D%3D&trackingId=iWJzZNhFRxyZCXWN5wOeQw%3D%3D
If that is not ok to post @mwetter please feel free to take it down.

**Reply by lambtt** [reply]
Hi, nice to meet you and really thanks for sharing!
